src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -10,12 +10,12 @@ import { listTemplate } from '../ListTemplate';
 import {
   DEFAULT_JOB_LOG_COLUMNS,
   DEFAULT_JOB_LOG_COLUMNS_WIDTHS,
-  FILE_STATUSES,
   checkboxListShape,
   getJobLogsListColumnMapping,
 } from '../../utils';
 
 import sharedCss from '../../shared.css';
+import { statusCellFormatter } from '../../utils/formatStatusCell';
```

Comment:
Move utils import before styles import, please

You can find the rules here - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&